providers/base/units/audio/jobs.pxu

```diff
@@ -369,26 +369,28 @@ _description:
 plugin: shell
 category_id: com.canonical.plainbox::audio
 id: audio/detect_sinks
-flags: also-after-suspend
+flags: also-after-suspend fail-on-resource
```

Comment:
After the long discussion we had on mattermost, I think it's better to remove this `fail-on-resource` flag. The reason is that if we put it, the audio jobs are going to fail for all the devices where the tester specified `manifest.has_audio_playback: false`